content/Visibility/Visibility.facts.md

```diff
@@ -0,0 +1,56 @@
+### Facts for Visibility lesson:
+
+Visibility defines the accessibility of a property and a method of a certain class. 
+
+Visibility keywords are defined as the following:
+
+ - `public` sets the accessibility of a property and a method of a class everywhere.
```

Comment:
Something isn't quite right with the phrasing here. Maybe "public sets the accessibility of a property or method such that it can be **accessed** by code inside or outside of the class"?